commands/build.go

```diff
@@ -926,3 +940,29 @@ func maybeJSONArray(v string) []string {
 	}
 	return []string{v}
 }
+
+func recordVersionInfo(mp metric.MeterProvider, command string) {
+	if !isExperimental() {
```

Comment:
nit: add extra comment in here saying that we are still in the process of testing/stabilizing these counters.